src/main/java/step3/domain/LottosWinningStatistics.java

```diff
@@ -0,0 +1,74 @@
+package step3.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottosWinningStatistics {
+    private final static int THREE_MATCHED_COUNT = 3;
+    private final static int FOUR_MATCHED_COUNT = 4;
+    private final static int FIVE_MATCHED_COUNT = 5;
+    private final static int SIX_MATCHED_COUNT = 6;
+    private final static int NOT_MATCHED_REWARD = 0;
+    private final static int THREE_MATCHED_REWARD = 5000;
+    private final static int FOUR_MATCHED_REWARD = 50000;
+    private final static int FIVE_MATCHED_REWARD = 1500000;
+    private final static int SIX_MATCHED_REWARD = 2000000000;
```

Comment:
```suggestion
    private final static int SIX_MATCHED_REWARD = 2_000_000_000;
```

천단위가 넘어가는 숫자는 '_' 를 사용하면 가독성을 챙길 수 있습니다 :)

위 상수 목록들을 `enum` 으로 정의해서 관리하는것도 고려해볼 수 있을 것 같네요 ㅎㅎ